src/coreclr/src/jit/emitarm64.cpp

```diff
@@ -12066,6 +12074,7 @@ void emitter::emitDispIns(
             printf("[");
             if (id->idAddr()->iiaIsJitDataOffset())
             {
+                printf("[");
```

Comment:
What is this for? Won't we end up with `[[...]`?

Reply:
Yeah, that is the left over code of something that I was trying. Will remove it.